roles/rachel/tasks/rachel_enabled.yml

```diff
@@ -1,3 +1,4 @@
+# FIXME
```

Comment:
```suggestion
```
roles/rachel has been deprecated for years.

As far as I know, we don't want to fix it, as /library/www/html/modules works as is (enabled in other ways).